packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -314,19 +321,32 @@ class ScrollDragController implements Drag {
         _offsetSinceLastStop = 0.0;
       }
       // Not moving can't break threshold.
-      return false;
+      return 0.0;
     } else {
       if (_offsetSinceLastStop == null) {
-        // Already in motion. Allow transparent offset transmission.
-        return true;
+        // Already in motion or no threshold behavior configured.
+        // Allow transparent offset transmission.
+        return offset;
```

Comment:
Can you add some comments in this method that talk about how by default, Android falls down one path and iOS the other, and point to the class that sets things up that way? Thanks.